### PR TITLE
Plot CT using HU and provide window presets

### PR DESCRIPTION
**Pull request by Haydnspass**
Followup pull requests (https://github.com/e0404/matRad/pull/122) because different branch is used.
CT windows from CERR (https://github.com/adityaapte/CERR).

### CODE REVIEW

**Comment by markbangert** [comment]
Looks good but we should also modify the DICOM import accordingly. Upon import I would store both HU and eDens. That way we do not need a back conversion and do not run into problem with non bijective HLUTs. Could you please add this functionality?

**Comment by Haydnspass** [comment]
That should done in (modified) line 64 in calcWaterEqD already?

**Comment by markbangert** [comment]
Argh. Didn't get that in the first place. Will have a thorough look at the code tomorrow and hopefully merge.

File: dicomImport/matRad_calcWaterEqD.m [hunk 1]
```diff
@@ -32,61 +32,18 @@
 %
 % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+% set conversion to HU to EqD if conversionMode is not provided
+% if (~exist('conversionMode', 'var'))
+%     conversionMode = HU2EqD;
+% end
+
```
Comment (markbangert) [inline dicomImport/matRad_calcWaterEqD.m]: What is this? This variable is never used and also never passed to the function -->the if statement is always true.

**Pull request merged**